tests/search/test_vespasearch.py

```diff
@@ -857,3 +857,100 @@ def test_all_data_download(data_db, data_client):
         "https://cdn.climatepolicyradar.org/"
         "navigator/dumps/whole_data_dump-2024-03-22.zip"
     )
+
+
+@pytest.mark.search
+def test_simple_pagination_families__temp_page_size_check(
+    test_vespa, data_client, data_db, monkeypatch
+):
+    monkeypatch.setattr(search, "_VESPA_CONNECTION", test_vespa)
+    _populate_db_families(data_db)
+
+    LIMIT = 2
+
+    # Query one
+    params = {
+        "query_string": "and",
+        "page_size": LIMIT,
+        "offset": 0,
+    }
+    body_one = _make_search_request(data_client, params)
+    assert body_one["hits"] == VESPA_FIXTURE_COUNT
+    assert len(body_one["families"]) == LIMIT
+    assert (
+        body_one["families"][0]["family_slug"]
+        == "agriculture-sector-plan-2015-2019_7999"
```

Comment:
Just wondering whether these can be parameterised i.e. so if the test data changes these auto update?

Reply:
Theres been a bit of back and forth on this, they where originally how you described. Probably best to leave as is for now and maybe consider a bigger test refactor